kubetest2-gke/deployer/up.go

```diff
@@ -223,7 +231,7 @@ func (d *Deployer) createCommand() []string {
 	return fs
 }
 
-func (d *Deployer) createWindowsNodePoolCommand(project string, cluster cluster, locationArg, nodePoolName, imageType string) []string {
+func (d *Deployer) createNodePoolCommand(project string, cluster cluster, locationArg, nodePoolName, imageType string, machineType string, numNodes int) []string {
 	fs := make([]string, 0)
 	fs = append(fs, "container", "node-pools", "create", nodePoolName)
```

Comment:
does this block until the node pool returns healthy or do we need to do additional validation?